webapp-client/src/stylesheets/ContentPane.css

```diff
@@ -0,0 +1,3 @@
+.wrap {
+	flex-wrap: wrap;
+}
```

Comment:
this file is no longer needed and removed on master, so you can either remove it here or resolve it during merge:)